modules/sd-planning-considerations.adoc

```diff
@@ -68,6 +70,26 @@ GCP control plane and infrastructure node size
 |custom-16-131072-ext
 |===
 
+GCP control plane and infrastructure node size for clusters created on or after 6 June 2024.
+[options="header",cols="2a,2a,2a"]
+|===
+|Number of compute nodes
```

Comment:
There should be a colon after each example sentence (I think). See https://spaces.redhat.com/display/CCSENABLE/Colons+and+semicolons ("Use a colon after the label of a note.") Thoughts?

Number of compute nodes:
Control plane size:
Infrastructure node size:

Reply:
Hi @obrown1205,
    Hope all is well! A bit confused by what you are asking/suggesting: Are you saying we should have a colon after each one of the sentences (shown in the attached image)? I don't think we should have them after the header titles. 
But, you bring something to my attention regardless. I put a period after the sentence I contributed to the docs, and the two leading sentences for the tables above do not have any punctuation, so I do need to match those sentences. Maybe add a colon to all three? 
  
![image](https://github.com/openshift/openshift-docs/assets/122639474/a98209b1-b0d4-4c93-8b4d-82b4d74c68f3)

Reply:
Yep! Sorrty - that's what I was referring to. Added the note to the wrong sentences :D